app/src/androidTest/java/com/github/h3lp3rs/h3lp/GuideTest.kt

```diff
@@ -0,0 +1,77 @@
+package com.github.h3lp3rs.h3lp
+
+import android.content.Context
+import androidx.preference.PreferenceManager
+import androidx.test.core.app.ApplicationProvider
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.assertion.ViewAssertions
+import androidx.test.espresso.assertion.ViewAssertions.matches
+import androidx.test.espresso.matcher.ViewMatchers
+import androidx.test.espresso.matcher.ViewMatchers.isDisplayed
+import androidx.test.espresso.matcher.ViewMatchers.withText
+import androidx.test.ext.junit.rules.ActivityScenarioRule
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import androidx.test.platform.app.InstrumentationRegistry
+import androidx.test.uiautomator.UiDevice
+import androidx.test.uiautomator.UiSelector
+import junit.framework.TestCase.assertFalse
+import junit.framework.TestCase.assertTrue
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4::class)
+class GuideTest {
+    @get:Rule
+    val testRule = ActivityScenarioRule(
+        MainPageActivity::class.java
+    )
+
+    private val targetContext : Context = ApplicationProvider.getApplicationContext()
+
+    private fun clearPreferences() {
+        val sharedPrefs = PreferenceManager.getDefaultSharedPreferences(targetContext)
```

Comment:
Depending on the merge order, the use of the PreferenceManager might have to be switched to call the local storage (though this should be a simple fix)